package.json

```diff
@@ -54,7 +54,7 @@
     "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
+    "release": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
```

Comment:
If you have an npm script named `publish`, lerna will try to run it during `lerna publish` lifecycle. Renaming script to release to avoid this.